lib/project-data-updater.js

```diff
@@ -22,8 +22,7 @@ module.exports.isValidUpdate = function(info) {
 module.exports.fileUpdated = function(commits) {
   for (var i = 0; i != commits.length; i++) {
     var commit = commits[i];
-    if ((commit.added.indexOf(module.exports.ABOUT_YML) !== -1) ||
-      (commit.modified.indexOf(module.exports.ABOUT_YML) !== -1)) {
+    if (commit.added.length + commit.modified.length > 0) {
```

Comment:
Ok, but that'll require a little more rearchitecting, as it currently assumes that it either has an error or that it has processed an .about.yml file. Workin' on it...